### PR TITLE
[NVIDIA TF] Enable BF16 TopK

### PR DESCRIPTION
**Pull request by trevor-m**
Adds GPU bfloat16 support for TopK.

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @rainwoodman Can you please review this PR ? Thank you!

**Review by cantonios — APPROVED** [review]

File: tensorflow/python/kernel_tests/math_ops/topk_op_test.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def _validateTopK(self,
         # Do some special casing of equality of indices: if indices
         # are not the same, but values are floating type, ensure that
         # the values are within epsilon of each other.
-        if not np.issubdtype(np_expected_values.dtype, np.floating):
+        if not np.issubdtype(np_expected_values.dtype, np.floating) and \
```
Comment (cantonios) [inline tensorflow/python/kernel_tests/math_ops/topk_op_test.py:58]: Sorry, can you remove the `\` and use brackets instead?  This is against our style guides.  See [here](https://google.github.io/styleguide/pyguide.html#32-line-length).  Thanks.

**Review by cantonios — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@trevor-m Can you please check @cantonios's comments and keep us posted ? Thank you!

File: tensorflow/python/kernel_tests/math_ops/topk_op_test.py (hunk 1 above)
Comment (trevor-m) [reply tensorflow/python/kernel_tests/math_ops/topk_op_test.py]: Thanks @cantonios, fixed it.

**Review by cantonios — APPROVED** [review]

**Comment by gbaned** [comment]
Seems auto-merge is not happening but the changes are merged into master now, so we can close this. Thank you for the PR.

**Pull request closed without merging**